README.md

```diff
@@ -1,6 +1,8 @@
 ### Far Manager
 https://www.farmanager.com
 
+[![AppVeyor](https://img.shields.io/appveyor/ci/FarGroup/farmanager.svg)](https://ci.appveyor.com/project/FarGroup/farmanager/history)
```

Comment:
What is img.shields.io?
I see different examples on appveyor page:

![badges](https://user-images.githubusercontent.com/11453922/36245670-0ea5258e-1224-11e8-8df7-34cd2e5cc8aa.png)